src/Model/Route.php

```diff
@@ -1,23 +1,42 @@
 <?php
 
+/**
+ * Copyright - elgentos ecommerce solutions (https://elgentos.nl)
+ */
+
+declare(strict_types=1);
+
 namespace Elgentos\PrismicIO\Model;
 
 use Elgentos\PrismicIO\Api\Data\Route\StoreInterface;
+use Elgentos\PrismicIO\Api\Data\RouteInterface;
 use Elgentos\PrismicIO\Model\ResourceModel\Route\Store\Collection;
 use Elgentos\PrismicIO\Model\ResourceModel\Route\Store\CollectionFactory;
 use Magento\Framework\Data\Collection\AbstractDb;
+use Magento\Framework\DataObject\IdentityInterface;
+use Magento\Framework\Model\AbstractModel;
 use Magento\Framework\Model\Context;
 use Magento\Framework\Model\ResourceModel\AbstractResource;
 use Magento\Framework\Registry;
 
-class Route extends \Magento\Framework\Model\AbstractModel implements \Elgentos\PrismicIO\Api\Data\RouteInterface, \Magento\Framework\DataObject\IdentityInterface
+class Route extends AbstractModel implements RouteInterface, IdentityInterface
 {
-    const CACHE_TAG = 'elgentos_prismicio_route';
-    const STORE_IDS = '_store_ids';
+    private const CACHE_TAG = 'elgentos_prismicio_route';
```

Comment:
This is a breaking change. And it might even be used by some logic to determine what cache to flush. Please don't change its visibility.